### PR TITLE
Improve test coverage

### PR DESCRIPTION
**Pull request by lei-Qiao**

### CODE REVIEW

File: tests/test_backend.py [hunk 1]
```diff
@@ -2352,7 +2359,7 @@ def test_softsign(self):
 
     def test_batch_to_spacend(self):
         block_size = [2, 2]
-        crop = [[0, 1], [2, 1]]
+        crop = [[1, 0], [2, 1]]
```
Comment (zhijxu-MS) [inline tests/test_backend.py]: what is the purpose of this change?

Comment (lei-Qiao) [reply tests/test_backend.py]: top, bottom = crops[0] 
        left, right = crops[1] 
        starts = [top, left] 
        ends = [] 
        for end in [bottom, right]: 
            if end != 0: 
                ends.append(-end) 
            else: 
                ends.append(np.iinfo(np.int32).max)

to cover the case end == 0

File: tf2onnx/onnx_opset/tensor.py [hunk 2]
```diff
@@ -1258,10 +1258,8 @@ def version_10(cls, ctx, node, **kwargs):
         utils.make_sure(seq_dim is not None, "sequence dim must be given in {}".format(node.name))
         seq_dim = seq_dim.i
         batch_dim = node.get_attr("batch_dim")
-        if batch_dim is not None:
-            batch_dim = batch_dim.i
-        else:
-            batch_dim = 0
+        #batch_dim is set by default 0 in tf
+        batch_dim = batch_dim.i
```
Comment (lucienwang1009) [inline tf2onnx/onnx_opset/tensor.py]: node.get_attr_value?

File: tf2onnx/onnx_opset/tensor.py [hunk 3]
```diff
@@ -1257,9 +1257,7 @@ def version_10(cls, ctx, node, **kwargs):
         seq_dim = node.get_attr("seq_dim")
         utils.make_sure(seq_dim is not None, "sequence dim must be given in {}".format(node.name))
         seq_dim = seq_dim.i
-        batch_dim = node.get_attr("batch_dim")
-        #batch_dim is set by default 0 in tf
-        batch_dim = batch_dim.i
+        batch_dim = node.get_attr_value("batch_dim")
```
Comment (lucienwang1009) [inline tf2onnx/onnx_opset/tensor.py]: hm... the default of `get_attr_value` is None, set it to yours.
